postman/adept.postman_collection.json

```diff
@@ -0,0 +1,1039 @@
+						"variables": ""
+					}
+				},
+				"url": {
+					"raw": "{{url}}",
+					"host": [
+						"{{url}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "search skills",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "6606f46f-3cb1-411f-a06a-382d87dee789",
+						"exec": [
+							"pm.test(\"200 status code\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "graphql",
+					"graphql": {
+						"query": "query Query($searchSkillsName: String) {\n  searchSkills(name: $searchSkillsName) {\n    skillId\n    name\n  }\n}",
+						"variables": "{\n  \"searchSkillsName\": \"React\"\n}"
+					}
+				},
+				"url": {
+					"raw": "{{url}}",
+					"host": [
+						"{{url}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "add skill to user",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "55b73123-7c51-408e-8723-2096f1033eba",
+						"exec": [
+							"pm.test(\"200 status code\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "graphql",
+					"graphql": {
+						"query": "mutation Mutation($addSkillToUserUserId: ID!, $addSkillToUserSkillId: ID!) {\n  addSkillToUser(userId: $addSkillToUserUserId, skillId: $addSkillToUserSkillId) {\n    name\n    userId\n    skills {\n      name\n      skillId\n    }\n  }\n}",
+						"variables": "{\n  \"addSkillToUserUserId\": \"{{userId}}\",\n  \"addSkillToUserSkillId\": \"{{skillId}}\"\n}"
+					}
+				},
+				"url": {
+					"raw": "{{url}}",
+					"host": [
+						"{{url}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "add contact to user",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "9b620125-a592-4837-aed1-bbb6d6b1f916",
+						"exec": [
+							"pm.test(\"200 status code\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "graphql",
+					"graphql": {
+						"query": "mutation Mutation($addContactToUserUserId: ID!, $addContactToUserContactId: ID!) {\n  addContactToUser(userId: $addContactToUserUserId, contactId: $addContactToUserContactId) {\n    name\n    email\n    contacts {\n      name\n      email\n      userId\n    }\n  }\n}",
+						"variables": "{\n  \"addContactToUserUserId\": \"{{userId}}\",\n  \"addContactToUserContactId\": \"{{userId2}}\"\n}"
+					}
+				},
+				"url": {
+					"raw": "{{url}}",
+					"host": [
+						"{{url}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "add education to user",
```

Comment:
I'm getting this error on the 'add education to user' test when trying to run all the tests in postman:

```
{"errors":[{"message":"Cannot read property 'create' of undefined","locations":[{"line":2,"column":3}],"path":["addEducationToResume"],"extensions":{"code":"INTERNAL_SERVER_ERROR","exception":{"stacktrace":["TypeError: Cannot read property 'create' of undefined","    at addEducationToResume (/Users/devin/Code/projects/adept/adept-graphql-server/src/schema/resolvers.js:342:48)","    at processTicksAndRejections (internal/process/task_queues.js:97:5)"]}}}],"data":null}
```

`addEducationToResume` resolver is returning null for some reason


When I try to delete the user using deleteUser in the GraphQL playground to rerun the tests, I get this error:

```
"message": "\nInvalid `prisma.resume.update()` invocation:\n\n\n  The table `School` does not exist in the current database."
```

This is line 208 in `resolvers.js`. I didn't think I would need to run any prisma commands to get the latest version of the database since we don't run the database locally. I feel like it has something to do with changing the type from School to Education. Let me know if I need to run any database updates other than having the updated schema.

I also think we should be merging development into main, not the feature branch into main. Lemme know if I'm wrong.

I'm also not sure if maintaining postman tests on github is the best option. Every time the postman collection changes, we need to reimport it in postman when we want to run the updated tests. If we as a team want to use postman tests, I think they have a shared collection feature we can use instead of github. If this is just for the grader, then that's fine but we can just submit the collections through Canvas.